client/webserver/site/src/css/order.scss

```diff
@@ -0,0 +1,70 @@
+div.order-datum {
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+  justify-content: start;
+  text-align: center;
+  border: 1px solid #777a;
+  font-size: 15px;
+  margin: 5px 10px 5px 0;
+
+  div:first-child {
+    padding: 2px 15px;
+    border-bottom: 1px solid #777a;
+    font-family: $demi-sans;
+    background-color: #7772;
+  }
+
+  div:last-child {
+    padding: 8px 15px;
```

Comment:
I'm finding 8px top/bottom padding for these a bit much.  Here's 4px:

![image](https://user-images.githubusercontent.com/9373513/92494313-2dee6d00-f1bb-11ea-813a-32561149396b.png)

vs 8px

![image](https://user-images.githubusercontent.com/9373513/92494389-46f71e00-f1bb-11ea-8679-093a2097f68c.png)

It's very subjective ofc, but I feel like 8 makes these a bit clunky in appearance, but perhaps there's another styling trick to improve that.